### ISSUE TITLE
Request: See all bands for given carrier on same map

### ISSUE THREAD

**Issue by brycenesbitt** [CellMapper/Cellmapper-Site#40]
When locating a tower for T-Mobile USA, I find myself flipping from GSM to UMTS to LTE a lot, because
it most cases the same tower serves all three and and there's useful information.

How about a mode that shows all available coverage for a carrier, with each each band as a color gradient:

- LTE Red to Dark Red
- UMTS Blue to Dark Blue
- GSM Yellow to Dark Yellow
- CDMA Gray to Black

See also http://blog.usabilla.com/how-to-design-for-color-blindness/

This would make it much easier to find towers.

**Comment by olkitu** [comment]
Current map not support that and T-Mobile must limit request count because example mobile browser will crash. Also it's too messy have too many markers on same map. I think today people want find LTE towers...  2G/3G not well accurated like LTE...

**Comment by brycenesbitt** [comment]
Many towers are multi carrier,
and almost all towers are multi band.

Having all the data helps a lot finding towers.  Right now it's a lot of flipping back and forth on maps: mobile or desktop.

**Comment by olkitu** [comment]
Yes, mostly towers have 2G/3G/4G but current design does not allow that. Also 2G/3G towers cannot be well located because there is no standard like 4G have.

**Comment by brycenesbitt** [comment]
The big problem with CDMA mapping in CellMapper is that CDMA tower ID's are
not globally unique.
Thus cellmapper attaches signal readings to a tower that may be hundreds of
miles away.

CDMA is still relevant in the USA, because Verizon has widely spaced towers.

------------------

Still:
If I am mapping a tower on any band, seeing the other bands is very very
very useful in finding the tower.
